x-pack/plugins/lens/public/embeddable/embeddable_component.tsx

```diff
@@ -47,16 +54,28 @@ type LensAttributes<TVisType, TVisState> = Omit<
  * Type-safe variant of by value embeddable input for Lens.
  * This can be used to hardcode certain Lens chart configurations within another app.
  */
-export type TypedLensByValueInput = Omit<LensByValueInput, 'attributes'> & {
+export type TypedLensByValueInput = Omit<LensByValueInput, 'attributes' | 'overrides'> & {
   attributes:
     | LensAttributes<'lnsXY', XYState>
     | LensAttributes<'lnsPie', PieVisualizationState>
+    | LensAttributes<'lnsHeatmap', HeatmapVisualizationState>
+    | LensAttributes<'lnsGauge', GaugeVisualizationState>
     | LensAttributes<'lnsDatatable', DatatableVisualizationState>
     | LensAttributes<'lnsLegacyMetric', LegacyMetricState>
     | LensAttributes<'lnsMetric', MetricVisualizationState>
-    | LensAttributes<'lnsHeatmap', HeatmapVisualizationState>
-    | LensAttributes<'lnsGauge', GaugeVisualizationState>
     | LensAttributes<string, unknown>;
+
+  /**
+   * Overrides can tweak the style of the final embeddable and are executed at the end of the Lens rendering pipeline.
+   * XY charts offer an override of the Settings ('settings') and Axis ('axisX', 'axisLeft', 'axisRight') components.
+   * While it is not possible to pass function/callback/handlers to the renderer, it is possible to stop them by passing the
+   * "ignore" string as override value (i.e. onBrushEnd: "ignore")
```

Comment:
`onBrushEnd` is included in https://github.com/elastic/kibana/pull/152842, no? So, is this really a limitation?